### PR TITLE
Fill food fields

### PR DESCRIPTION
**Pull request by adrian-lara**
Link to Pivotal Tracker ticket:
https://www.pivotaltracker.com/story/show/154615068

Additional comments:
@sidewinder2020 
This isn't a very long commit, but it might be easier to start at HTML, the index.js, the Event Listener, and the Response Handler.

Forgot to mention this in person, if you end up approving PR, could you "Accept" the PivotalTracker story (which marks the ticket done)? We can use a different process if you'd prefer; we can discuss in person.

### CODE REVIEW

File: lib/response-handlers/foods-response-handler.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const postNewFood = (name, calories) => {
+  let newFood = { food: {'name': name, 'calories': calories}}
+
+  let postOptions = {
+    method: "POST",
+    body: JSON.stringify(newFood),
+    headers: {'Content-Type': 'application/json'},
+  }
+  fetch(`${url}/api/v1/foods`, postOptions)
```
Comment (sidewinder2020) [inline lib/response-handlers/foods-response-handler.js:13]: I don't know if I agree with this fetch in the response handlers file? I think the code is clean, but I'm not sure I understand the refactor.  I can ask you about it later :)

File: lib/response-handlers/foods-response-handler.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const url = `https://protected-basin-11627.herokuapp.com/`
+
+const handleResponse = require('../response-handlers/initial-response-handler')
```
Comment (sidewinder2020) [inline lib/response-handlers/foods-response-handler.js:3]: It looks like you have an initial-response-handler file w/in the response handlers folder.  I understand where you're trying to go with this, but I want to see how it plays out.  I worry that the code might become too dry

File: lib/event-listeners/foods-event-listeners.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const postNewFood = require('../response-handlers/foods-response-handler').postNewFood
+
+const newFoodSubmit = () => {
+  $('#new-food-submit-button').on("click", (event) => {
+    event.preventDefault(event)
+
+    let name = $('#new-food-form [name=new-food-name').val()
```
Comment (sidewinder2020) [inline lib/event-listeners/foods-event-listeners.js:7]: well structured event listener :)

**Review by sidewinder2020 — APPROVED** [review]

File: lib/response-handlers/foods-response-handler.js (hunk 2 above)
Comment (adrian-lara) [reply lib/response-handlers/foods-response-handler.js:3]: I was thinking it'd be useful for both of us to have this available within our respective pages. I don't mind extracting this out and putting it back into this response handler file. But for now, I'm also leaning towards seeing how things go when the app becomes more built out.

If at any time we feel like we're getting close to a level organization either of us are uncomfortable with, let's definitely let each other know and talk about it 😄 👍

File: lib/response-handlers/foods-response-handler.js (hunk 1 above)
Comment (adrian-lara) [reply lib/response-handlers/foods-response-handler.js:13]: Ah, actually, I meant to abstract this into foods-fetch-requests. I'll move it in a soon-coming PR 😃 

Nice catch!

**Pull request merged**